<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DicomExplorer</class>
 <widget class="QMainWindow" name="DicomExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DICOM Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeWidget" name="headerTree">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="traceTree">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="matplotlibFig" name="mplWindow" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="openButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="quitButton">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>matplotlibFig</class>
   <extends>QWidget</extends>
   <header>matplotlibfig.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>DicomExplorer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1030</x>
     <y>729</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>DicomExplorer</receiver>
   <slot>onOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1030</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpen()</slot>
 </slots>
</ui>
